src/main/app/controller/courts/LocalAuthoritiesController.ts

```diff
@@ -0,0 +1,193 @@
+import autobind from 'autobind-decorator';
+import {AuthedRequest} from '../../../types/AuthedRequest';
+import {Response} from 'express';
+import {CourtType} from '../../../types/CourtType';
+import {
+  LocalAuthoritiesAreaOfLaw,
+  LocalAuthoritiesPageData,
+  LocalAuthority,
+  LocalAuthorityItem
+} from '../../../types/LocalAuthority';
+import {AreaOfLaw} from '../../../types/AreaOfLaw';
+
+import {SelectItem} from '../../../types/CourtPageData';
+import {CSRF} from '../../../modules/csrf';
+
+export enum familyAreaOfLaw {
+  adoption = 'Adoption',
+  children = 'Children',
+  civilPartnership = 'Civil partnership',
+  divorce = 'Divorce'
+}
+@autobind
+export class LocalAuthoritiesController {
+
+
```

Comment:
Think there are a few places in the code where there may be more than one space, might be worth making it so there is only one space

Reply:
removed all spaces